src/pages/instances/InstanceList.tsx

```diff
@@ -585,35 +586,38 @@ const InstanceList: FC = () => {
               )}
               {!hasInstances && (
                 <EmptyState
-                  iconName="containers"
-                  iconClass="empty-instances-icon"
```

Comment:
I think the `empty-instances-icon` class is now unused. Can you remove it from `styles.scss` as well?
Also check the other icon classes that were removed in other files.